addons/grenades/stringtable.xml

```diff
@@ -144,6 +144,12 @@
             <Korean>플래시뱅이라고도 알려져있습니다. 사용즉시 섬광으로 인한 시력장애, 청각장애, 이명, 내이기관방해를 유발합니다.</Korean>
             <Chinesesimp>也被称为闪光弹，会造成暂时性失明，耳聋，耳鸣等效果。</Chinesesimp>
             <Chinese>也被稱為閃光彈，會造成暫時性失明，耳聾，耳鳴等效果</Chinese>
+        </Key>
+		<Key ID="STR_ACE_Grenades_CTS9_Name">
+            <English>CTS 7290-9 9-bang Flash Grenade</English>
+        </Key>
+        <Key ID="STR_ACE_Grenades_CTS9_NameShort">
+            <English>CTS 7290-9</English>
```

Comment:
Couldn't this also be used for all the other languages?

Reply:
Yes.

Reply:
Need to change displayName from 7290-9 to 7290. The 9-bang part is duplicate otherwise. And the name looks crap in arsenal